### PR TITLE
[NG] Changed <Link> to <a> in SocialLinks component

### LINKED ISSUE #1220: [NG] Use <a> tags instead of <Link>
**Desired Situation**
In [SocialLinks](https://github.com/layer5io/layer5/blob/layer5-ng/src/components/SocialLinks/index.js) component use `<a>` tags instead of `<Link>` for all the links 

---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by thadanipaarth**
**Description**
Changed the `<Link>` to `<a>` in SocialLinks component

This PR fixes #1220 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/components/SocialLinks/index.js [hunk 1]
```diff
@@ -14,27 +14,48 @@ const SocialLinks = () => {
         <SocialSectionWrapper>
             <Col className="social-widgets" xs={12}>
                 <Row className="contactf">
-                    <Link className="social-icons mail" href="mailto:community@layer5.io">
+                    <a 
+                        className="social-icons mail" 
+                        href="mailto:community@layer5.io" 
+                        target="_blank"
```
Comment (Tanuj22) [inline src/components/SocialLinks/index.js:20]: Also, add this in all the <a> tags ` rel="noreferrer"`

File: src/components/SocialLinks/index.js [hunk 2]
```diff
@@ -14,27 +14,48 @@ const SocialLinks = () => {
...
+                    >
                         <img height="40 px" src={mail_icon} alt="mail" />
-                    </Link>
-                    <Link className="social-icons" to="http://slack.layer5.io/">
+                    </a>
```
Comment (Tanuj22) [inline src/components/SocialLinks/index.js:23]: Why is this here?

File: src/components/SocialLinks/index.js [hunk 3]
```diff
@@ -14,27 +14,48 @@ const SocialLinks = () => {
...
+                    <a 
+                        className="social-icons" 
+                        href="https://twitter.com/layer5" 
+                        target="_blank"
+                    >
                         <img className="twitter" height="40 px" src={twitter_icon} alt="twitter" />
-                    </Link>
-                    <Link className="social-icons" to="https://github.com/layer5io">
+                    </a>
```
Comment (Tanuj22) [inline src/components/SocialLinks/index.js:37]: Same

File: src/components/SocialLinks/index.js [hunk 4]
```diff
@@ -14,27 +14,48 @@ const SocialLinks = () => {
...
+                    <a 
+                        className="social-icons" 
+                        href="https://github.com/layer5io" 
+                        target="_blank"
+                    >
                         <img className="github" height="40 px" src={github_icon} alt="github" />
-                    </Link>
-                    <Link
+                    </a>
```
Comment (Tanuj22) [inline src/components/SocialLinks/index.js:44]: Same

File: src/components/SocialLinks/index.js [hunk 5]
```diff
@@ -14,27 +14,48 @@ const SocialLinks = () => {
...
+                    <a
                         className="social-icons"
-                        to="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1"
+                        href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1"
+                        target="_blank"
                     >
                         <img className="youtube" height="40 px" src={youtube_icon} alt="youtube" />
-                    </Link>
-                    <Link className="social-icons" to="https://hub.docker.com/u/layer5/">
+                    </a>
```
Comment (Tanuj22) [inline src/components/SocialLinks/index.js:51]: Same

**Review by Jashpatel1 — COMMENTED** [review]
Nice work here @thadanipaarth . Just a couple of changes:
Remove unused Link imports.
Add `rel="noreferrer"` in every anchor tag, to pass build checks.

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Thanks for also opening it on a new page. Some minor changes and we are good to go :) .
Also remove the unsed `import { Link } from "gatsby";` line

**Comment by thadanipaarth** [comment]
Noticed the unwanted import later, Have fixed it. Also, added the `rel="noreferrer"` for fixing the security issue. Have a look!

**Review by Jashpatel1 — APPROVED** [review]
Looks good. Cool work, here. Thanks @thadanipaarth

**Review by Tanuj22 — APPROVED** [review]
Nice work!
